index.html

```diff
@@ -49,129 +54,272 @@ <h4>I’m a Software Developer</h4>
-                        <button class="btn" type="submit">See project</button>
                     </div>
-                </div>
-                <div class="cards-container">
-                    <div class="card-container">
-                        <h3 class="title-card">Profesional Art Printing Data</h3>
-                        <div class="desc">
-                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                    <div class="cards-container">
+                        <div class="card-container">
+                            <h3 class="title-card">Profesional Art Printing Data</h3>
+                            <div class="desc">
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                            </div>
+                            <div class="row-card-skills">
+                                <ul>
+                                    <li>HTML</li>
+                                    <li>Bootstrap</li>
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                            <button class="btn" type="submit">See project</button>
                         </div>
-                        <div class="row-card-skills">
-                            <ul>
-                                <li>HTML</li>
-                                <li>Bootstrap</li>
-                                <li>Ruby</li>
-                            </ul>
+                    </div>
+                    <div class="cards-container">
+                        <div class="card-container">
+                            <h3 class="title-card">Profesional Art Printing Data</h3>
+                            <div class="desc">
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                            </div>
+                            <div class="row-card-skills">
+                                <ul>
+                                    <li>HTML</li>
+                                    <li>Bootstrap</li>
+                                    <li>Ruby</li>
+                                </ul>
+                            </div>
+                            <button class="btn" type="submit">See project</button>
                         </div>
-                        <button class="btn" type="submit">See project</button>
                     </div>
                 </div>
+                <!-- SECTION FOR MOBILE PROJECTS GRID-ROW END HERE -->
+
             </section>
+            <!-- SECTION FOR THE MAIN END HERE -->
 
+            <!-- SECTION ABOUT BASE START HERE -->
             <section class="about base">
                 <div class="title title-about">
```

Comment:
- [ ] **[OPTIONAL]** You did a great job on your about me section! However, I noticed that there is too much space **at the top of the section**, which is causing it to deviate from the given [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=vzFkfnAyKEgehx5P-0). To improve the overall look of your project and **make it more consistent with the template, you might consider adjusting the spacing in this section.** Keep up the good work! 🤓

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236360879-28ba79e5-acad-4e52-8227-1365775f3333.png)

  ### Figma template:
  ![image](https://user-images.githubusercontent.com/107952724/236360903-7ce16a2d-b7ae-40a2-8720-fba9bd570440.png)